frontend/package.json

```diff
@@ -31,6 +31,7 @@
 		"@ant-design/icons": "4.8.0",
 		"@grafana/data": "^8.4.3",
 		"@monaco-editor/react": "^4.3.1",
+		"@types/react-highlight-words": "^0.16.4",
```

Comment:
By the way, why don't use `yarn.lock`?

Sometimes it may lead to a merge mess, but exact versions don't solve the problem. Transitive dependencies are still with `^`.

Reply:
>Sometimes it may lead to a merge mess, but exact versions don't solve the problem. Transitive dependencies are still with ^

can you share more about it...